src/jquery.floatingscroll.js

```diff
@@ -134,8 +143,8 @@
                 cont = inst.cont,
                 pos = cont.getBoundingClientRect();
             inst.sbar.width($(cont).outerWidth());
-            if (!inst.scrollBody) {
-                inst.sbar.css("left", pos.left + "px");
+            if (inst.scrollContainer || !inst.scrollBody) {
```

Comment:
Could you please explain why we need to modify this condition? Let's suppose that the user has added both the `.fl-scrolls-container` element and the `.fl-scrolls-viewport`+`.fl-scrolls-body` couple. In this case we may get wrong positioning of the scroll bar as its left position is being relative to the `.fl-scrolls-viewport` element while `pos.left` is relative to the viewport.
Here is a demo: https://jsfiddle.net/oxpqpvh7/
And here is the same demo but with old variant of the condition: https://jsfiddle.net/7odr9yte/